packages/next-wxt/env/.env.open-prod

```
@@ -1,7 +1,9 @@
+VITE_MODEL_CONFIG=open
 VITE_LLM_BASE_URL=https://agent.opentiny.design/api/v1/ai/prompt
 VITE_LLM_API_KEY=sk-trial
 VITE_LLM_MODEL=deepseek-ai/DeepSeek-V3
 VITE_LLM_PROVIDER_TYPE=deepseek
 VITE_WEB_AGENT_CONNECT_TYPE=mcp
 VITE_AGENT_ROOT=https://agent.opentiny.design/api/v1/webmcp-trial/
-VITE_ROBOT_URL=https://chat.opentiny.design
\ No newline at end of file
+VITE_ROBOT_URL=https://chat.opentiny.design
+VITE_TOKEN_API_URL=xxx
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)